docs/index.html

```diff
@@ -25,14 +25,16 @@
         </div>
     </div>
     <div class="main-container">
+      <div class="cardHolder">
         <p class="h1 text-center">Recipe Champion</p>
-        <p id="lead" class="lead text-center">Enter your ingredients and we'll help you find the perfect recipe</p>
+        <p id="lead" class="leader text-center">Enter your ingredients and we'll help you find the perfect recipe</p>
```

Comment:
Is there a specific reason to change this?
The joke prepends to "lead" so either update the .js to match or keep it lead.

Reply:
the id lead and class lead were conflicting. I'll update JS